yb-voyager/src/adaptiveparallelism/adaptive_parallelism.go

```diff
@@ -22,29 +22,39 @@ import (
 
 	"github.com/davecgh/go-spew/spew"
 	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 )
 
 const (
-	CPU_USAGE_USER                 = "cpu_usage_user"
-	CPU_USAGE_SYSTEM               = "cpu_usage_system"
-	MAX_CPU_THRESHOLD              = 70
-	ADAPTIVE_PARALLELISM_FREQUENCY = 10 * time.Second
+	CPU_USAGE_USER   = "cpu_usage_user"
+	CPU_USAGE_SYSTEM = "cpu_usage_system"
 )
 
+var MAX_CPU_THRESHOLD int
+var ADAPTIVE_PARALLELISM_FREQUENCY_SECONDS int
+
+func init() {
+	MAX_CPU_THRESHOLD = utils.GetEnvAsInt("MAX_CPU_THRESHOLD", 70)
+	ADAPTIVE_PARALLELISM_FREQUENCY_SECONDS = utils.GetEnvAsInt("ADAPTIVE_PARALLELISM_FREQUENCY_SECONDS", 10)
+}
+
 type TargetYugabyteDBWithConnectionPool interface {
 	IsAdaptiveParallelismSupported() bool
-	GetClusterMetrics() (map[string]map[string]string, error) // node_uuid:metric_name:metric_value
+	GetClusterMetrics() (map[string]tgtdb.NodeMetrics, error) // node_uuid:metric_name:metric_value
```

Comment:
@priyanshi-yb because of this struct, i had to add dependency on tgtdb. Spent some time thinking of whether there's a way to avoid it, but could not come up with anything reasonable. 

1. I could create an interface for the NodeMetrics struct, with getters and setters, but that seems like an anti-pattern - https://github.com/golang/go/issues/23796#issuecomment-365040949. 
2. On second thoughts, I don't see a big problem with adaptive parallelism depending on tgtdb. It sort of acts on top of it. In fact your earlier suggestion of just putting it in tgtdb also speaks to that. 

But for now, it's reasonable to structure it this way: i.e. have a separate package for adaptive parallelism, depend on tgtdb for some structs, depend on an interface for yugabyteDB (to allow for easier testing). 

Let me know your thoughts.